Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+            "sponsorship_id": contract.sponsorship_id,
+            "creator_id": contract.creator_id,
+            "brand_id": contract.brand_id,
+            "contract_url": contract.contract_url,
+            "status": contract.status,
+            "created_at": t
+        }).execute()
+        
+        if response.data:
+            return response.data[0]
+        else:
+            raise HTTPException(status_code=400, detail="Failed to create contract")
```

Ruff (0.13.3)

Abstract `raise` to an inner function

(TRY301)